packages/react-core/src/Authenticator/hooks/useAuthenticatorComponents/__tests__/useAuthenticatorComponent.spec.ts

```diff
@@ -0,0 +1,29 @@
+import { renderHook } from '@testing-library/react-hooks';
+import { capitalize } from '@aws-amplify/ui';
+
+import { RenderNothing } from '../../../../components';
+import { COMPONENT_ROUTE_KEYS } from '../../constants';
+import { DEFAULTS } from '../__mock__/components';
+
+import { useAuthenticatorComponents } from '..';
+
+describe('useAuthenticatorComponents', () => {
```

Comment:
From the hook implementation it seems it can take in overrides, can we add a test for that as well? Bonus covering an unhappy path scenario and verify it is handled gracefully.

Reply:
The unhappy path test scenarios are covered by the utility that handles them. IMO do not need redundant tests but more than happy to discuss